core/templates/pages/story-viewer-page/story-viewer-page.mainpage.html

```diff
@@ -77,6 +77,37 @@
         width: 100px;
       }
 
+      .lesson-icon-completed {
+        position: absolute;
```

Comment:
Alphabetize the properties here and below.

Reply:
Done.